src/openrct2-ui/windows/TitleEditor.cpp

```diff
@@ -936,6 +936,26 @@ static void window_title_editor_scrollpaint_commands(rct_window *w, rct_drawpixe
             set_format_arg(0, uintptr_t, name);
             break;
         }
+        case TITLE_SCRIPT_LOADSC:
+        {
+            commandName = STR_TITLE_EDITOR_COMMAND_LOAD_FILE;
+            const char * name = "";
+            source_desc desc;
+            if (command->Scenario[0] == '\0')
+            {
+                commandName = STR_TITLE_EDITOR_COMMAND_LOAD_NO_SCENARIO;
+            }
+            else
+            {
+                auto scenario = GetScenarioRepository()->GetByFilename(command->Scenario);
+                if (scenario != nullptr && scenario_get_source_desc(scenario->name, &desc))
```

Comment:
`ScenarioSources::TryGetByName`?

Reply:
`TryGetByName` attempts to locate the title sequence based on it's translated title. This won't work when sharing title sequences between different languages.

Reply:
https://github.com/OpenRCT2/OpenRCT2/blob/00fd18809ca71e4b73e8d3c4d658d2139d5dd881/src/openrct2/scenario/ScenarioSources.cpp#L361-L364

Reply:
It looks like it won't work if I use it based on the scenario name even if it isn't language based. At the moment, there's no function to convert from scenario name to scenario path. That and scenarios in RCT1 are named sc1.SC4 and not their actual name. Again, it stores the scenario as it's filepath.